scrapers/fl/bills.py

```diff
@@ -141,6 +134,34 @@ def process_page(self):
         yield HouseSearchPage(self.input)
         yield from self.process_votes()
 
+    def process_sponsors(self):
+        sponsor = self.root.xpath(
+            'string(//div[@id="main"]/div/div/p[span[contains(text(),"by")]])'
+        ).strip()
+
+        SPONSOR_RE = re.compile(r"by\s+(?P<sponsors>[^(]+)(\(CO-INTRODUCERS\)\s+(?P<cosponsors>[\s\S]+))?")
```

Comment:
Given we're compiling this regex without any variables, can we move it outside the function so we only compile it on import instead of every call? It isn't a _huge_ change, but it may make things a bit faster.